answers/chapter11.md

```diff
@@ -0,0 +1,33 @@
+  1. Multithreading allows you to partition programs into work units/threads which can run in parallel. 
+  This can make your program more efficient on a multi processor system, or when a thread is waiting for resources etc (another thread can run in that time). 
+  
+  2. Multithreading is supported by the Thread class and the Runnable interface
+  
+  3. You would extend Thread if you wanted to override a method of thread _other than_ `run()` 
+  
+  4. Erm? Surely the object would be called `myThread`? Shrug:
+  
+  ```java
+  MyThrd.join()`;
+  ```
+  
+  5. ```java
+  thread.setPriority(Thread.NORM_PRIORITY+3);
+  ```
+  
+  6. A synchronized method only allows one thread to access that method _or any other synchronized method in the same class_. 

```

Comment:
I found out about the second part after searching for clues on how to answer part 12.